collect_app/src/main/java/org/odk/collect/android/activities/GoogleSheetsUploaderActivity.java

```diff
@@ -392,7 +394,12 @@ public void uploadingComplete(HashMap<String, String> result) {
         StringBuilder message = new StringBuilder();
 
         if (keys.size() == 0) {
-            message.append(getString(R.string.no_forms_uploaded));
+            if(mAuthFailed) {
+                message.append(getString(R.string.auth_failed_form_upload));
+                mAuthFailed = false;
+            }
+            else
```

Comment:
Please follow [this](https://google.github.io/styleguide/javaguide.html) style guide (I just made sure it's added to the contributing guide). The else should be on the same line as the closing brace and there should be braces around all statements.